Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -760,13 +763,21 @@ def create_text_out_format(iocs: IO, request_args: RequestArguments) -> Union[IO
             new_line = '\n'
     iocs.close()
     if len(ipv4_formatted_indicators) > 0:
+        if extensive_logging:
+            demisto.debug(f"edl: Original IPv4 indicators: {ipv4_formatted_indicators}")
         ipv4_formatted_indicators = ips_to_ranges(ipv4_formatted_indicators, request_args.collapse_ips)
+        if extensive_logging:
+            demisto.debug(f"edl: Formatted IPv4 indicators: {ipv4_formatted_indicators}")
         for ip in ipv4_formatted_indicators:
             formatted_indicators.write(new_line + str(ip))
             new_line = '\n'
 
     if len(ipv6_formatted_indicators) > 0:
+        if extensive_logging:
+            demisto.debug(f"edl: Original IPv6 indicators: {ipv6_formatted_indicators}")
```

Comment:
the same addition of longline of length like the comments above

Reply:
done